gulpfile.js

```diff
@@ -30,24 +28,26 @@ gulp.task('copy', function() {
       .pipe(gulp.dest('built/'));
 });
 
-gulp.task('clang', function() {
-  return gulp.src(['src/**/*.ts'])
-      .pipe(gulpFormat.checkFormat('file', clangFormat))
-      .on('warning', function(e) {
-    console.log(e);
-  });
+gulp.task('format:enforce', () => {
+  const format = require('gulp-clang-format');
```

Comment:
I copied this from [the `webdriver-manager` gulp file](https://github.com/angular/webdriver-manager/blob/master/gulpfile.js#L34), which I think @cnishina copied from someone else.  I'd be happy to move it.